eng/pipelines/runtime-staging.yml

```diff
@@ -518,11 +518,41 @@ jobs:
       # Turn off the testing for now, until https://github.com/dotnet/runtime/issues/60128 gets resolved
       # ${{ if eq(variables['isRollingBuild'], true) }}:
       #   # extra steps, run tests
-      #   extraStepsTemplate: /eng/pipelines/common/templates/runtimes/android-runtime-and-send-to-helix.yml
+      #   extraStepsTemplate: /eng/pipelines/common/templates/runtimes/build-runtime-tests-and-send-to-helix.yml
       #   extraStepsParameters:
       #     creator: dotnet-bot
       #     testRunNamePrefixSuffix: Mono_$(_BuildConfig)
 
+#
+# Special Configurations - Rolling build only
+#
+
+#
+# Build the whole product using Mono for Android and run runtime tests with Android devices
+# Co-op GC 
```

Comment:
Is this comment wrong? or am I misunderstanding the YAML? this is going to build/run on linux-x64, right? not Android?

Reply:
I didn't edit the comment.  I was thinking of running the runtime tests, but then changed it.